### DISCUSSION TITLE
VScode compatibility

### DISCUSSION THREAD

**Discussion by mocenigo** [zed-industries/zed#8892] [Feature Requests]
Hi, I am sure this must have been asked somewhere already. Does Zed have the goal to be (somewhat) compatible with VSCode extensions? Not all expensions would run, I guess especially those that need to rely on electron directly. But it would still be a great help to adoption if that were possible. The source code is very clean and readable and I expect that this way zed could become an editor "for the ages" (like immortal emacs and vi).

**Comment by mikayla-maki** [comment]
We're doing our own thing with WASM based extensions. So if they can be compiled to WASM, and executed on `wasmtime`, then we could theoretically support them. But this seems unlikely, and we'll almost certainly be exposing different APIs anyway.
